src/term/mod.rs

```diff
@@ -1411,23 +1411,33 @@ impl ansi::Handler for Term {
     fn put_tab(&mut self, mut count: i64) {
         trace!("put_tab: {}", count);
 
-        let mut col = self.cursor.point.col;
-        while col < self.grid.num_cols() && count != 0 {
+        while self.cursor.point.col < self.grid.num_cols() && count != 0 {
             count -= 1;
+
+            {
+                let cell = &mut self.grid[&self.cursor.point];
+                *cell = self.cursor.template;
+                cell.c = self.cursor.charsets[self.active_charset].map('\t');
```

Comment:
If `cell.c` is going to be a `'\t'`, I think the rendered needs to be updated to handle it like a `' '`

Reply:
Ok, I think I have the copy code referencing `self.tabs`. Two questions. 

1. `self.tabs` looks like it could use an abstraction around lookup by column because everything else is also by column. Mind if I add one?
2. The renderer is rendering tabs as 0 height & width glyph, like a space. So, at least on my box, it's being handled fine. Maybe leave the renderer alone until there's a problem?